woodwork/logical_types.py

```diff
@@ -22,7 +22,7 @@ class LogicalType(object, metaclass=LogicalTypeMetaClass):
     type_string = ClassNameDescriptor()
     pandas_dtype = 'string'
     backup_dtype = None
-    standard_tags = {}
+    standard_tags = set()
```

Comment:
Was causing issues that this was a dictionary by default since we return standard tags in `_reset_semantic_tags`